.circleci/config.yml

```diff
@@ -56,6 +56,52 @@ jobs:
             git config --local user.email "sysadmin@binary.com"
             git config --local user.name "gh-pages deploy bot"
             make gh-pages
+  update_schema:
+    <<: *default
+    steps:
+      - checkout
```

Comment:
the add_ssh_keys step is missing here. Do we need that to be able to push the updated schema to master ?

Reply:
I think we should use a user deploy key to push to master. 
I'm not sure the token in that `add_ssh_keys` step in docs-build-deploy has permission to write origin master or not.